src/librustc_driver/driver.rs

```diff
@@ -515,9 +515,7 @@ pub fn phase_2_configure_and_expand(sess: &Session,
     time(time_passes, "gated macro checking", || {
```

Comment:
This isn't doing gated macro checking anymore. Also, I don't think `syntax::feature_gate::get_features` needs its own timed pass since it is doing so little.
Perhaps we could time a single task consisting of `get_features` and the preceding three tasks (`update_recursion_limit`, `compute_crate_disambiguator`, and `collect_crate_types`, the latter two of which are not currently timed)?